sdgym/benchmark.py

```diff
@@ -691,6 +678,15 @@ def _run_jobs(multi_processing_config, job_args_list, show_progress):
         raise SDGymError('No valid Dataset/Synthesizer combination given.')
 
     scores = pd.concat(scores, ignore_index=True)
+    output_directions = job_args_list[0][-1]
+    if output_directions and isinstance(output_directions, dict):
+        synth_path = Path(output_directions['synthesizer'])
+        root_path = synth_path.parents[2]
+        result_file = root_path / 'results.csv'
+        if not result_file.exists():
+            scores.to_csv(result_file, index=False, mode='w')
+        else:
+            scores.to_csv(result_file, index=False, mode='a', header=False)
```

Comment:
Can we write to a different file here as well to assure no collisions? I was also thinking for the run id that we could just make it the date followed by a number that we increment. Like 20251202_1 or something like that. That might be more readable than a uuid